assets/RVC/Cloud/Applio Colab.md

```diff
@@ -33,8 +33,8 @@ order: 5000
 - Automatic model upload
 - Has Mangio-Crepe
 - User-friendly UI
+- It is the fastest compared to all other forks

```

Comment:
not objective, there are no independent statistics on that so nope

Reply:
I don't understand this, what do you need a graph to show you which Applio is the fastest? Before **it wasn't objective**, now it is.
![man](https://github.com/AIHubCentral/docs/assets/67287120/b910571f-8532-4163-bcec-32f340243a9d)

Reply:
I have never said anything on this before, and the graph doesn't show much information therefore I don't think it is good to promote something that can't be proven. Also the docs should display an objective opinion on everything and the suggested edits are clearly biased.